docs/content/docs/tutorials/emr-on-eks-example.md

```diff
@@ -0,0 +1,368 @@
+aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+helm install --create-namespace -n $ACK_SYSTEM_NAMESPACE ack-$SERVICE-controller \
+  oci://public.ecr.aws/aws-controllers-k8s/$SERVICE-chart --version=$RELEASE_VERSION --set=aws.region=$AWS_REGION
+```
+**Expected outcome**
+```
+NAME: ack-emrcontainers-controller
+LAST DEPLOYED: Fri Aug 26 09:05:08 2022
+NAMESPACE: ack-system
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+emrcontainers-chart has been installed.
+This chart deploys "public.ecr.aws/aws-controllers-k8s/emrcontainers-controller:v0.0.6".
+
+Check its status by running:
+  kubectl --namespace ack-system get pods -l "app.kubernetes.io/instance=ack-emrcontainers-controller"
+
+You are now able to create Amazon EMR on EKS (EMRContainers) resources!
+```
+#### Configure IRSA for emr on eks controller
+Once the controller is deployed, you need to setup IAM permissions for the controller so that it can create and manage resources using EMR, S3 and other API's. We will use [IAM Roles for Service Account](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to secure this IAM role so that only EMR on EKS controller service account can assume the permissions assigned.
+Please follow [how to configure IAM permissions](https://aws-controllers-k8s.github.io/community/docs/user-docs/irsa/) for IRSA setup. Make sure to change the value for `**SERVICE**` to `**emrcontainers**`
+
+After completing all the steps, please validate annotation for service account before proceeding.
+```
+# validate annotation
+kubectl get pods -n $ACK_K8S_NAMESPACE
+echo "sleeping for 5sec for pod to get ready"
+sleep 5
+CONTROLLER_POD_NAME=$(kubectl get pods -n $ACK_K8S_NAMESPACE --selector=app.kubernetes.io/name=emrcontainers-chart -o jsonpath='{.items..metadata.name}')
+kubectl describe pod -n $ACK_K8S_NAMESPACE $CONTROLLER_POD_NAME | grep "^\s*AWS_"
+```
+**Expected outcome**
+```
+AWS_REGION:                      us-west-2
+AWS_ENDPOINT_URL:                
+AWS_ROLE_ARN:                    arn:aws:iam::012345678910:role/ack-emrcontainers-controller
+AWS_WEB_IDENTITY_TOKEN_FILE:     /var/run/secrets/eks.amazonaws.com/serviceaccount/token (http://eks.amazonaws.com/serviceaccount/token)
+```
+
+## Create EMR VirtualCluster
+We can now create EMR Virtual Cluster. An EMR Virtual Cluster is mapped to a Kubernetes namespace. EMR uses virtual clusters to run jobs and host endpoints.
+```
+cat << EOF > virtualcluster.yaml
+---
+apiVersion: emrcontainers.services.k8s.aws/v1alpha1
+kind: VirtualCluster
+metadata:
+  name: my-ack-vc
+spec:
+  name: my-ack-vc
+  containerProvider:
+    id: $EKS_CLUSTER_NAME
+    type_: EKS
+    info:
+      eksInfo:
+        namespace: emr-ns
+EOF
+```
+Let's create a virtualcluster
+```
+envsubst < virtualcluster.yaml | kubectl apply -f -
+kubectl describe virtualclusters
+```
+**Expected outcome**
```

Comment:
Not sure but if VirtualClusters take some time to create, maybe mention that it might now be instantly synced?

Reply:
Its pretty fast in my observations.